SPECS/cri-o/cri-o.spec

```diff
@@ -0,0 +1,1713 @@
+
+%install
+cd $HOME/go/src/%{project}
+
+# Binaries
+install -D -m 0755 bin/crio    %{buildroot}/%{_bindir}/crio
+install -D -m 0755 bin/crio-status    %{buildroot}/%{_bindir}/crio-status
+install -D -m 0755 bin/pinns    %{buildroot}/%{_bindir}/pinns
+install -d %{buildroot}/%{_libexecdir}/crio/bin
+# Completions
+install -D -m 0644 completions/bash/crio %{buildroot}/%{_datadir}/bash-completion/completions/crio
+install -D -m 0644 completions/zsh/_crio %{buildroot}%{_sysconfdir}/zsh_completion.d/_crio
+install -D -m 0644 completions/fish/crio.fish %{buildroot}/%{_datadir}/fish/completions/crio.fish
+install -D -m 0644 completions/bash/crio-status %{buildroot}/%{_datadir}/bash-completion/completions/crio-status
+install -D -m 0644 completions/zsh/_crio-status %{buildroot}%{_sysconfdir}/zsh_completion.d/_crio-status
+install -D -m 0644 completions/fish/crio-status.fish %{buildroot}/%{_datadir}/fish/completions/crio-status.fish
+# Manpages
+install -d %{buildroot}/%{_mandir}/man5
+install -d %{buildroot}/%{_mandir}/man8
+install -m 0644 docs/crio.conf.5 %{buildroot}/%{_mandir}/man5
+install -m 0644 docs/crio.8      %{buildroot}/%{_mandir}/man8
+# Configs
+sed -e 's-@LIBEXECDIR@-%{_libexecdir}-g' -i %{SOURCE4}
+install -D -m 0644 %{SOURCE4}       %{buildroot}/%{_sysconfdir}/crio/crio.conf.d/00-default.conf
+install -D -m 0644 crio-umount.conf %{buildroot}/%{_datadir}/oci-umount/oci-umount.d/cri-umount.conf
+install -D -m 0644 %{SOURCE3}       %{buildroot}%{_fillupdir}/sysconfig.crio
+# Systemd
+install -D -m 0644 %{SOURCE2} %{buildroot}%{_unitdir}/crio.service
+# place kubelet.env in fillupdir
+install -D -m 0644 %{SOURCE6} %{buildroot}%{_fillupdir}/sysconfig.kubelet
+# Symlinks to rc files
+install -d -m 0755 %{buildroot}%{_sbindir}
+ln -sf service %{buildroot}%{_sbindir}/rccrio
+
+%fdupes %{buildroot}/%{_prefix}
+
+%files
+# Binaries
+%{_bindir}/crio
+%{_bindir}/crio-status
+%{_bindir}/pinns
+%dir %{_libexecdir}/crio
+%dir %{_libexecdir}/crio/bin
+# Completions
+%{_datadir}/bash-completion/completions/crio
+%{_datadir}/bash-completion/completions/crio-status
+%{_sysconfdir}/zsh_completion.d
+%{_sysconfdir}/zsh_completion.d/_crio
+%{_sysconfdir}/zsh_completion.d/_crio-status
+%{_datadir}/fish
+%{_datadir}/fish/completions
+%{_datadir}/fish/completions/crio.fish
+%{_datadir}/fish/completions/crio-status.fish
+# Manpages
+%{_mandir}/man5/crio.conf.5*
+%{_mandir}/man8/crio.8*
+# License
+%license LICENSE
+# Configs
+%dir %{_sysconfdir}/crio
+%dir %{_sysconfdir}/crio/crio.conf.d
+%config %{_sysconfdir}/crio/crio.conf.d/00-default.conf
+%dir %{_datadir}/oci-umount
+%dir %{_datadir}/oci-umount/oci-umount.d
+%{_datadir}/oci-umount/oci-umount.d/cri-umount.conf
+%{_fillupdir}/sysconfig.crio
+# Systemd
+%{_unitdir}/crio.service
+%{_sbindir}/rccrio
+
+%files kubeadm-criconfig
+%defattr(-,root,root)
+%{_fillupdir}/sysconfig.kubelet
+
+%changelog
+* Thu Aug 19 2021 Henry Li <lihl@microsoft.com> - 1.21.2-2
+- Initial CBL-Mariner import from OpenSUSE Tumbleweed
```

Comment:
Good news! We've recently figured out the openSUSE licensing for specs. Please use this from now on:
```
- Initial CBL-Mariner import from OpenSUSE Tumbleweed (license: same as "License" tag).
```

Please also update `libcontainers-common`.